deploy/cluster_role.yaml

```diff
@@ -15,7 +15,13 @@ rules:
   - configmaps
   - secrets
   verbs:
-  - '*'
+  - create
```

Comment:
Note to reviewers: Expanding these verbs per the [docs](https://sdk.operatorframework.io/docs/upgrading-sdk-version/version-upgrade-guide/#v013x) -- see also https://github.com/operator-framework/operator-sdk/issues/671